onnxruntime/core/framework/execution_frame.cc

```diff
@@ -282,8 +282,8 @@ ExecutionFrame::ExecutionFrame(const std::vector<int>& feed_mlvalue_idxs, const
             }
 
             // log size of activation. Keep it commented out for now to avoid log flooding.
-            // VLOGS(session_state_.Logger(), 1) << "Allocated memory for activations, size: "
-            //                                   << mem_patterns_->patterns[i].PeakSize();
+            std::cout << "Allocated memory for activations, size: "
+                      << mem_patterns_->patterns[i].PeakSize() << "\n";
```

Comment:
will revert

Reply:
seems not reverted yet

---
In reply to: [468117231](https://github.com/microsoft/onnxruntime/pull/4526#discussion_r468117231) [](ancestors = 468117231)